hypothesis-python/tests/nocover/test_cache_implementation.py

```diff
@@ -73,8 +73,9 @@ def set_key(self, key):
 
     @invariant()
     def check_values(self):
-        for k in getattr(self, "cache", ()):
-            assert self.__values[k] == self.cache[k]
+        self.cache.check_valid()
```

Comment:
A curiosity. When this method asserted, it did not report the falsifying example and stateful steps. Apparently as a consequence of internal error escalation, since the check happens in a hypothesis file.

Just checking, this is as-intended @Zac-HD? Do we have an established pattern to selectively not escalate in our own tests?

Reply:
It's not particularly intended, but also doesn't bother me much and so I don't think we (yet) have a pattern to avoid it - no objection if you'd like to add one though.